package.json

```diff
@@ -24,7 +25,8 @@
     "storage-inspect:generate": "./script/storage-check.sh generate ZoraCreator1155Impl ZoraCreator1155FactoryImpl",
     "release": "yarn run prepack && changeset publish",
     "js-test:watch": "vitest dev",
-    "anvil": "anvil --fork-url https://rpc.zora.energy --fork-block-number 2550000 --chain-id 31337"
+    "anvil": "anvil --fork-url https://rpc.zora.energy --fork-block-number 2550000 --chain-id 31337",
+    "update-version": "changeset version && yarn update-contract-version"
```

Comment:
Looked and didn't see a simpler way to integrate the call into pre-release

Reply:
you could do it in the `prepack` step, but i thought it should happen only whenever the version is updated.